### PR TITLE
MAW-23: Added a navbar for review

### PR DESCRIPTION
**Pull request by yeboah326**
Please review my initial submission for the navbar

### CODE REVIEW

File: src/styles/navbar.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    font-size: 30px;
+    color: orange;
+}
+
+.nav-items{
+    height: 80%;
+    width: 60%;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
```
Comment (pktwum96) [inline src/styles/navbar.css:28]: Space around nav item components too large
image

File: src/styles/navbar.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
+@import url('https://fonts.googleapis.com/css2?family=Doppio+One&display=swap');
+
+.navbar{
+    height: 10vh;
+    width: 100%;
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-around;
+    font-family: 'Doppio One', 'sans-serif';
+    border-bottom: 1px solid orange;
```
Comment (pktwum96) [inline src/styles/navbar.css:10]: Why? lol

File: src/styles/navbar.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
+@import url('https://fonts.googleapis.com/css2?family=Doppio+One&display=swap');
+
+.navbar{
+    height: 10vh;
+    width: 100%;
```
Comment (pktwum96) [inline src/styles/navbar.css:5]: Please create a JIRA ticket to create a responsive component for the navbar

File: src/styles/navbar.css [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    justify-content: center;
+}
+
+.nav-item:hover{
+    border-bottom: 3px solid orange;
+}
+
+.nav-item-selected{
+    border-bottom: 3px solid orange;
+}
```
Comment (pktwum96) [inline src/styles/navbar.css:47]: Please run prettier to fix linting issues

**Comment by pktwum96** [comment]
Looks good and I like the direction you are headed! Please reference the JIRA ticket in your Pull Request titles

**Review by pktwum96 — CHANGES_REQUESTED** [review]

File: src/styles/navbar.css (hunk 4 above)
Comment (yeboah326) [reply src/styles/navbar.css]: FIxed the linting errors

File: src/styles/navbar.css (hunk 1 above)
Comment (yeboah326) [reply src/styles/navbar.css:28]: I've reduced the space around the nav items

File: src/styles/navbar.css (hunk 2 above)
Comment (yeboah326) [reply src/styles/navbar.css:10]: This was the best way I could think of doing it. I'll find a better way to do it.

**Comment by yeboah326** [comment]
I've created a ticket for the responsive component of the navbar

**Review by pktwum96 — CHANGES_REQUESTED** [review]
Please rebase to pick up latest changes

**Comment by yeboah326** [comment]
Please i'm having some difficulties with the rebase. I'm getting conflicts with the navbar.css file and don't know whether to accept the incoming changes or use current changes.

**Pull request closed without merging**